### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zeibura**
Signed-off-by: Zeibura Kathau <zeibura.kathau@broadcom.com>

### CODE REVIEW

File: README.md
Comment (zimlu02) [inline README.md:30]: as agreed, let's wait for the HLASM release, that will come early next week.

**Review by zimlu02 — CHANGES_REQUESTED** [review]

**Review by jellypuno — APPROVED** [review]

File: README.md
Comment (IgorCATech) [reply README.md:113]: yes, that's a good suggestion actually. Let me go ahead and change it

File: README.md
Comment (zeibura) [reply README.md:30]: @zimlu02, Dejan was suggesting that we add info on IDMS support to high-level descriptions of COBOL LS. What do you think of this? Would this PR be the right time to do it, or should we wait for the next COBOL release?

Comment (zeibura) [reply README.md:31]: nevermind - we will do this when we update Code4z next time after the COBOL release.

**Review by zimlu02 — APPROVED** [review]

**Comment by zeibura** [comment]
please merge https://github.com/BroadcomMFD/code4z/pull/32 before merging this PR

**Review by zimlu02 — CHANGES_REQUESTED** [review]
- please update also COBOL LS to 0.19.0 (will be released today)
- add CCF into What's new section, highlighting that this is newly added to the package

**Review by zimlu02 — APPROVED** [review]

**Pull request merged**
